packages/pwa-kit-create-app/scripts/create-mobify-app.js

```diff
@@ -939,7 +950,7 @@ const fetchAvailableAppExtensions = () => {
  * @param {*} answers
  * @param {*} param2
  */
-const runGenerator = async (
```

Comment:
No need to have `async` here actually. 

In fact, it was causing a weird issue where an error was not surfaced. An error inside the `runGenerator` function was strangely not caught at the end (see line 1314). 

So it looked as if everything was fine, and a successful message was printed out. This is probably why the failing project generation on Windows machine was not noticed until now.